src/main/java/com/bergerkiller/bukkit/tc/signactions/SignActionAnnounce.java

```diff
@@ -66,9 +71,29 @@ public boolean build(SignChangeActionEvent event) {
                 .setPermission(Permission.BUILD_ANNOUNCER)
                 .setName("announcer")
                 .setDescription(event.isRCSign() ?
-                        "remotely send a message to all the players in the train" :
-                        "send a message to players in a train")
+                        "announce a message to all the players in the train" :
```

Comment:
Right now it's not obvious from the sign placement message whether an RC sign or a normal sign is placed. This is not ideal.